scripts/api/data/dataset-create-new-with-type.json

```diff
@@ -0,0 +1,82 @@
+{
+  "datasetType": "testDatasetType",
+  "datasetVersion": {
+    "license": {
+      "name": "CC BY 4.0",
+      "uri": "https://creativecommons.org/licenses/by/4.0/"
+    },
+    "metadataBlocks": {
+      "citation": {
+        "fields": [
+          {
+            "value": "pyDataverse",
+            "typeClass": "primitive",
+            "multiple": false,
+            "typeName": "title"
+          },
+          {
+            "value": [
+              {
+                "authorName": {
+                  "value": "Range, Dan",
```

Comment:
Ha. Dan instead of Jan?

Reply:
Needed to make sure I had the right file for testing.